
package org.openspcoop2.core.tracciamento.ws.client.traccia.search;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.7.4
 * 2015-02-18T17:09:26.742+01:00
 * Generated source version: 2.7.4
 */

@WebFault(name = "tracciamento-multiple-result-exception", targetNamespace = "http://www.openspcoop2.org/core/tracciamento/management")
public class TracciamentoMultipleResultException_Exception extends Exception {
    
    /**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private org.openspcoop2.core.tracciamento.ws.client.traccia.search.TracciamentoMultipleResultException tracciamentoMultipleResultException;

    public TracciamentoMultipleResultException_Exception() {
        super();
    }
    
    public TracciamentoMultipleResultException_Exception(String message) {
        super(message);
    }
    
    public TracciamentoMultipleResultException_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public TracciamentoMultipleResultException_Exception(String message, org.openspcoop2.core.tracciamento.ws.client.traccia.search.TracciamentoMultipleResultException tracciamentoMultipleResultException) {
        super(message);
        this.tracciamentoMultipleResultException = tracciamentoMultipleResultException;
    }

    public TracciamentoMultipleResultException_Exception(String message, org.openspcoop2.core.tracciamento.ws.client.traccia.search.TracciamentoMultipleResultException tracciamentoMultipleResultException, Throwable cause) {
        super(message, cause);
        this.tracciamentoMultipleResultException = tracciamentoMultipleResultException;
    }

    public org.openspcoop2.core.tracciamento.ws.client.traccia.search.TracciamentoMultipleResultException getFaultInfo() {
        return this.tracciamentoMultipleResultException;
    }
}
